ChangeLog

```diff
@@ -14,6 +24,7 @@
 	* man/BermudanSwaption.Rmd: Modified to reflect DiscountCurve
 	option and example updated
 	
+>>>>>>> eddelbuettel/master
```

Comment:
I was hoping the screenshot would indicate my error. I suspect github desktop sync does not do what I think it does.Let me state the process as I see it with questions.Initially:1) Fork master (can be anywhere, so github fork ok, right?)2) clone it (did this originally via Studio project open of my github fork)3) write code, check in with local commit, check it with R CMD check4) Run Travis locally (how do I do this?)5) Happy with 4, push to fork on github6) Create pull checking files are correct (is there a way to delete files from a pull request?)So what if my repo is out of sync? Do I delete current forks and clones and redo steps 1& 2? When I did the sync, I thought it was creating a copy of Master, but I suspect I had some changes on my side that it worked into the repo. How do I get a clean new Master?